Re-enable py-torchtext for ml-darwin-aarch64-mps stack

zackgalbreath opened the pull request:
Fixed in https://github.com/spack/spack/pull/51011

zackgalbreath committed:
Re-enable py-torchtext for ml-darwin-aarch64-mps stack

Fixed in https://github.com/spack/spack/pull/51011

zackgalbreath commented:
FYI @kwryankrattiger @adamjstewart 

I'll rebase this on develop after #638 and #640 get merged.

adamjstewart commented:
Thanks for tracking this down! TorchText is semi-abandoned, so I'm not super concerned if we can't build it forever, but it's nice to build the same stack on all OSes.

zackgalbreath committed:
py-torch: relax version constraint for llvm-openmp with apple-clang

The bug that originally motivated this constraint was fixed in
https://github.com/spack/spack/pull/51011

zackgalbreath committed:
Revert "py-torch: relax version constraint for llvm-openmp with apple-clang"

This reverts commit e9fa4f23d3fd9f52fe01f9a46bde8fec56e3e668.

adamjstewart reviewed (approved):
Could still relax the constraint and apply only to older versions, but I'm not too worried.

zackgalbreath commented:
Closing in favor of #2331
